config/configure.in

```diff
@@ -563,6 +563,28 @@ else
   AC_MSG_RESULT([found $NUMS_INCLUDE])
 fi
 
+CURL=yes
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking ocaml curl library" >&5
+printf %s "checking ocaml curl library... " >&6; }
+CURL_INCLUDE="`ocamlfind query -i-format curl 2> /dev/null`"
+if test "$CURL_INCLUDE" == ""; then
+  CURL=no
+fi
```

Comment:
testing for empty outpust is slightly meh, should check exit code, but i guess it is not easy to do inside backticks

Reply:
Should be fixed.